production/ksonnet/promtail/config.libsonnet

```diff
@@ -10,7 +10,7 @@
         username:: '',
         password:: '',
         scheme:: 'https',
-        hostname:: 'logs-us-west1.grafana.net',
+        hostname:: 'set valid hostname here',
```

Comment:
This is what you are looking for. It could be a breaking change though for people updating their lib and promtails suddenly stopping to work.
```suggestion
        hostname:: error 'must define a valid hostname',
```

Reply:
Thanks @sandeepsukhani - that looks perfect!

Reply:
> It could be a breaking change though for people updating their lib and promtails suddenly stopping to work.

@sandeepsukhani If people are using this default setting, all they're getting is `404`s, so it's not working for them _anyway_ 😉